staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -118,6 +133,24 @@ type APIAggregator struct {
 	// handledGroups are the groups that already have routes
 	handledGroups sets.String
 
+	// Swagger spec for each api service
+	apiServiceSpecs map[string]*spec.Swagger
+
+	// List of the specs that needs to be loaded. When a spec is successfully loaded
+	// it will be removed from this list and added to apiServiceSpecs.
+	// Map values are retry counts. After a preset retries, it will stop
```

Comment:
Ugly. Put this into the variable name at the very least: `toLoadAPISpecRetries`.  Though, if a retry mechnism is really necessary, this ought to be in some separate controller instead of "globally" in the aggregator.